src/transforms.jl

```diff
@@ -228,4 +228,5 @@ include("transforms/quantile.jl")
 include("transforms/functional.jl")
 include("transforms/eigenanalysis.jl")
 include("transforms/sequential.jl")
-include("transforms/parallel.jl")
\ No newline at end of file
+include("transforms/parallel.jl")
+include("transforms/levels.jl")
```

Comment:
Same here, please include the file in a different place in the list. The sequential and parallel should be the last.